src/api/basic/transactions.ts

```diff
@@ -167,5 +175,5 @@ export type SwapResponse = {
   resultType: RoutingResultType
   route: QuoteSimulationResult | null
   error: string | null
-  tx: EvmTransaction | CosmosTransaction | SolanaTransaction | Transfer | null
+  tx: EvmTransaction | CosmosTransaction | SolanaTransaction | Transfer | StarknetTransaction | null
```

Comment:
It seems like `TronTransaction` is missing:
https://rango-api.readme.io/reference/swap